### ISSUE TITLE
Cannot reproduce the iwslt15 benchmark, just get bleu score ~8 

### ISSUE THREAD

**Issue by wb14123** [tensorflow/nmt#81]
Hi,

I've downloaded this project and try to reproduce the benchmark with iwslt15 dataset. Here is the command I've run:

```
python -m nmt.nmt \
--attention=scaled_luong \
--src=vi --tgt=en \
--vocab_prefix=/data/dl-data/iwslt15-google/vocab \
--train_prefix=/data/dl-data/iwslt15-google/train \
--dev_prefix=/data/dl-data/iwslt15-google/tst2012 \
--test_prefix=/data/dl-data/iwslt15-google/tst2013 \
--out_dir=./output \
--hparams_path=/home/wangbin/open_source/nmt/nmt/standard_hparams/iwslt15.json
```

I've changed `num_train_steps: 12000` from 12000 to 120000.

But I get the bleu score about 8 and it never increase. Here is the test bleu and training loss:

![screenshot_2017-08-24_11-33-24](https://user-images.githubusercontent.com/1906051/29648433-afd80158-88c0-11e7-9ee1-e45109afc829.png)
![screenshot_2017-08-24_11-34-10](https://user-images.githubusercontent.com/1906051/29648434-afe256a8-88c0-11e7-8698-9f6b24451094.png)

My TensorFlow version is v1.2 and the version of this project is b3f1bcc97caf388fc3cc3856fda8404e5c9e6f86 .

Is there something I was doing wrong? Thansk.

**Comment by cyzLoveDream** [comment]
hello, why can't  i open this web.and the train_log is messy code? can you help me?

**Comment by wb14123** [comment]
@cyzLoveDream This is the UI of [TensorBoard](https://www.tensorflow.org/get_started/summaries_and_tensorboard). Short story, use this command:

```
tensorboard --logdir <model_output_dir>
```

**Comment by bastings** [comment]
@wb14123 the benchmark is English to Vietnamese (not Vietnamese to English).

**Comment by cyzLoveDream** [comment]
@wb14123 thank you, i have run your command , but the website is nothing. this show it can't find events

**Comment by cyzLoveDream** [comment]
@bastings wow! i have run the code in python3, but the the events of train_log is messy code. this problem is because  i use the python3?

**Comment by wb14123** [comment]
@bastings I've retried with `--src=en --tgt=vi` but it seems to be fixed on bleu score ~11:

![screenshot_2017-08-24_18-04-53](https://user-images.githubusercontent.com/1906051/29661515-d73629c4-88f6-11e7-8763-42242de82fb3.png)

**Comment by wb14123** [comment]
@bastings Oh, I just found that you said it doesn't work in Python 3. I'll have a try with Python 2.

**Comment by wb14123** [comment]
Python 2 works for me. Closed this issue.

**Comment by mohammedayub44** [comment]
@bastings @oahziur  @lmthang - Is Python 3 still an issue for the current branch because I ran some experiments as I get bleu score =1.4 as max and it doesn't improve further. I thought it was fixed in some of the issues but I get very inconsistent results. A confirmation would ease my pain.
I'm using AWS Deep Learning AMI with Python 3.6. 

Thanks !

Mohammed Ayub
